interface/forms/newpatient/common.php

```diff
@@ -146,9 +146,6 @@ function refreshIssue(issue, title) {
                 e.preventDefault();
                 e.stopPropagation();
                 dlgopen('', '', 700, 650, '', '', {
-
-                    buttons: [{text: <?php echo xlj('Close'); ?>, close: true, style: 'default btn-sm'}],
```

Comment:
@stu01509 Dunno if you intended to delete this

Reply:
Was this meant to be deleted?

Reply:
this looks good on testing. see why this Close was removed now.

Reply:
Hi @bradymiller  and @tywrenn 

Ok, I undo this change.
if I remember correctly that modal has two close buttons that are the reason why I remove it.